### ISSUE TITLE
Need to create the default android webDir before/during the copy task

### ISSUE THREAD

**Issue by mlynch** [ionic-team/capacitor#818]
If you don't have `www` as your static folder when you run `npx cap add android`, then the `copy` task will try to `createEmptyCordovaJS` in Android's `webDir` which maps to `src/main/assets` which won't exist if the first copy failed.

To reproduce, run `npm init` on an app without a `www` folder and run `npx cap add android`. The fix would be creating the `webDir` when adding the platform, and making sure it's in `android-template`

**Comment by jcesarmobile** [comment]
I don't think this is a good idea, most times it happens to me is because it's an Ionic project and I forgot to generate the www folder, so creating a default one from the template will add files different from the user ones and they might be confused.

Instead, I think it should just show the error and not create the android folder
